test/node/conversation-guest-access.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const assert = require('assert');
+const Circuit = require('../../circuit-node');
+const config = require('./config.json');
+const helper = require('./helper');
+Circuit.logger.setLevel(Circuit.Enums.LogLevel.Error);
+
+let client;
+let user;
+let conversation;
+describe('Guest Access Tests', () => {
+    before(async () => {
+        client = new Circuit.Client(config.bot1);
+        user = await client.logon();
+    });
+
+    after(async () => {
+        await client.logout();
+    });
+
+    it('should create a conversation', async () => {
```

Comment:
we should not have a test to create a conversation here. either move that to the "before" or rename it to "should create a conference bridge" since a conf bridge is related to guest access and we can justify testing that here.

Reply:
done